### ISSUE TITLE
Work only in Wifi connection not 3G

### ISSUE THREAD

**Issue by BackPackerDz** [pchab/AndroidRTC#89]
Someone have the same problem ?

**Comment by DanielMor** [comment]
Yes, its a issue in libjingle for default use only wifi connections...

**Comment by BackPackerDz** [comment]
@DanielMor thank you for the reply

**Comment by CaoQuocTuan** [comment]
@BackPackerDz you can use turn server to connection with 3g.

**Comment by BackPackerDz** [comment]
@CaoQuocTuan how can I do it please ?

**Comment by sojharo** [comment]
I used to have this problem when I used my 4G device. This might be because service provider uses proxy or something. However, using TURN server solved my problem.

You can host your TURN server on separate machine. There many TURN implementations however, I often use coturn. This is available on Github and you can see the installation guide there too.

**Comment by PeterPan355** [comment]
@sojharo 
Hello Sojharo,
Can you please help me with this, because I really don't know how to connect to TURN server and make call.  How to implement connection to stun server from Android, or maybe you have some solution? Thank you.

**Comment by DanielMor** [comment]
@PeterPan355 you can use this turn project https://github.com/coturn/coturn

**Comment by alexruskovski** [comment]
What is the syntax for adding turn server?
